### PR TITLE
[Actions] Add greet action 🎉

### PR DESCRIPTION
**Pull request by Pranav-yadav**
## Summary:

Currently we do _not_ greet; **"First Time Contributors"**.
It is important to greet every first time contributor keep the OSS community thriving. 😊

This PR; Adds a greet action `greet.yml` :tada:, which comments under First Time Contributors PRs, greeting them as well suggesting some basic steps/guidelines.

## Changelog:

[GENERAL] [ADDED] - [Actions] Add greet action 🎉

## Test Plan:

- Should greet 1st time contributors

### CODE REVIEW

**Comment by Pranav-yadav** [comment]
@SlyCaptainFlint Take a look at this whenever you get time. (P.S. mentioned you since you recently refctored other actions)

Example of greet message (though, not added by this action):
https://github.com/facebook/react-native/pull/37488#issuecomment-1555709103


1. What else do we need to add in the greet message? (I would say; short, simple & sweet 😊)
2. Do we need to mention the contributor ? (FYI: This would add extra complexity)
3. What should be removed from current message?

Cc: @cortinico WDYT? 🙂

**Comment by Pranav-yadav** [comment]
This week we've seen a lot of **"First Time Contributors"**
Chain React 👍

Cc: @cipolleschi do you think having this action would benefit...?

**Comment by cortinico** [comment]
> 1. What else do we need to add in the greet message? (I would say; short, simple & sweet 😊)

The only thing I would add is a note to check what Danger is reporting, as people tend to ignore it. Other than that, I believe we're good to go 👍

File: .github/workflows/greet.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            Someone from the team will review your PR and get back to you _as soon as possible_ with feedback if any. :blush: Post review, and if all tests are passing, your PR will be _imported_ to internal FB phabricator and should be merged if it also passes all internal tests. :partying_face:
+            
+            <details>
+            <summary>P.S.: In the <i>meantime</i>, you should go through below checklist: <sup>(click to expand)</sup>
+            </summary>
+            
+            1. Make sure you have _signed_ the [CLA](https://code.facebook.com/cla) :balance_scale:,
+            2. There are _no_ typos :sweat_smile:,
+            3. All tests :test_tube: are _passing_. i.e. The CI checks should be green :green_circle:, and
+            4. You have _added_ necessary tests for your code, if it's a refactor or a new functionality. :text_tube:
```
Comment (Pranav-yadav) [inline .github/workflows/greet.yml:38]:
```suggestion
            4. You have _added_ necessary tests for your code, if it's a refactor or a new functionality. :test_tube:
```
🧪

**Comment by Pranav-yadav** [comment]
> > 1. What else do we need to add in the greet message? (I would say; short, simple & sweet 😊)
> 
> The only thing I would add is a note to check what Danger is reporting, as people tend to ignore it. Other than that, I believe we're good to go 👍

Done.
P.S. I'm not used to multiline strings syntax of yaml especially in actions so, may be you can validate; _if it'll work with proper formatting on GitHub_.

File: .github/workflows/greet.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    permissions:
+      contents: read
+      pull-requests: write # for actions/first-interaction to comment on PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1.1.1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: |
+            "### Hi there, :wave:
```
Comment (cipolleschi) [inline .github/workflows/greet.yml:23]: I think that, with the `|` operator, we can remove the `"` at the beginning and end of the string.

**Review by cipolleschi — APPROVED** [review]
Great job! I personally love that, as it may save us some repetitive messaging!

I left a small comment, which I am not super sure about too, tbh. XD

**Review by cortinico — APPROVED** [review]

**Comment by Pranav-yadav** [comment]
> I think that, with the | operator, we can remove the " at the beginning and end of the string.
>> Sure. Let's go with your suggestion. 😉👍
>> I hope the 1st contributor seeing this in action will be happy and most importantly, the msg formatting will be proper.

Done. 🚀

**Comment by Pranav-yadav** [comment]
@cipolleschi looks like you didn't import the latest changes, so the merged commit still has those `"`s 🙂

**Pull request closed without merging**